### DISCUSSION TITLE
rbenv install 2.5.1 not working

### DISCUSSION THREAD

**Discussion by okhan98** [rbenv/ruby-build#1859] [Build failures]
Whenever I try to install 2.5.1 , I get the following error.

2.5.1: line 1: syntax error near unexpected token `('
2.5.1: line 1: `ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin16]'

BUILD FAILED (Mac OS X 10.15.7 using ruby-build 20201225)

Any help would be appreciated. thanks!

**Comment by vldcreation** [comment]
are you done for this @okhan98 ?
